### PR TITLE
Implement insight page

### PR DESCRIPTION
**Pull request by cakecatz**
**What this PR does / why we need it**:

image

This screenshot uses dummy data.

Ref: https://github.com/pipe-cd/pipe/issues/1160

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
Add insight page
```

### CODE REVIEW

File: pkg/app/web/src/modules/deployment-frequency.test.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+describe("deploymentFrequencySlice reducer", () => {
+  it("should handle initial state", () => {
+    expect(
+      deploymentFrequencySlice.reducer(undefined, {
+        type: "TEST_ACTION",
+      })
+    ).toMatchInlineSnapshot(`
+      Object {
+        "data": Array [],
```
Comment (khanhtc1202) [inline pkg/app/web/src/modules/deployment-frequency.test.ts:11]: nit: should it be `[]` 🤔

Comment (cakecatz) [reply pkg/app/web/src/modules/deployment-frequency.test.ts:11]: This code is generated by `jest`, so we'll leave it as is.

Comment (khanhtc1202) [reply pkg/app/web/src/modules/deployment-frequency.test.ts:11]: I see 😅 thanks

**Comment by khanhtc1202** [comment]
/approve
🚀🚀🚀🚀🚀

**Pull request merged**